### ISSUE TITLE
Screen Reader Friendly can be determined in two ways that should be described.

### ISSUE THREAD

**Issue by w3cbot** [w3c/a11y-review#128]
**This is a tracker issue.** Only discuss things here if they are a11y group internal meta-discussions about the issue. **Contribute to the actual discussion at the following link:**

§ https://github.com/w3c/publ-a11y/issues/123
